### DISCUSSION TITLE
Count Records Error on Dart

### DISCUSSION THREAD

**Discussion by DevMensIT** [supabase/supabase#6332] [General]
I'm getting error here in my dart because of {count}

`supabase.from('table').select('*', { count: 'exact' })`

**Too many positional arguments: 1 expected, but 2 found.
Try removing the extra arguments.dart**

how can i get total records from table ?

**Comment by GaryAustin1** [comment]
Dart uses a different format.
https://supabase.com/docs/reference/dart/select#querying-with-count-option
